database/migrations/2021_08_26_004523_create_movie_celebrities_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+
+class CreateMovieCelebritiesTable extends Migration
```

Comment:
This table name should reflect the model and the controller, so if the model is celebritymodel then the migration should be create_celebrity_movies_table. 

The easiest fix is to create the migration again to make sure that the name is the same as the Model and Controller